src/_includes/docs/install/flutter-doctor.md

```diff
@@ -8,7 +8,7 @@
 {% case include.devos %}
 {% when 'Windows' -%}
    {% assign terminal='PowerShell' %}
-   {% assign prompt='C:\>' %}
+   {% assign prompt='PS C:\>' %}
```

Comment:
Why powershell?

Reply:
The instructions themselves always said you needed to install / use PowerShell, so I'm just making the prompting consistent.